cobigen/cobigen-textmerger/src/test/java/com/devonfw/cobigen/textmerger/TextAppenderTest.java

```diff
@@ -45,8 +46,9 @@ public void testMerge_appendWithoutNewLineNoAnchors() throws Exception {
     public void testMerge_appendWithNewLineNoAnchors() throws Exception {
         TextAppender appender = new TextAppender("textmerge_append", true);
         String mergedString = appender.merge(new File(testFileRootPath + "BaseFile.txt"), "Test3", "UTF-8");
+        mergedString = mergedString.replaceAll("\\n|\\r", "");
         assertThat(mergedString)
-            .isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile_withNewLine.txt")));
+            .isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile_withNewLine.txt")).replaceAll("\\n|\\r", ""));
```

Comment:
Also, as you use that regex multiple times, it would be better to create a new constant with a readable name like `ALL_LINE_ENDINGS`